jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeProperties.java

```diff
@@ -119,6 +119,7 @@ public class OdeProperties implements EnvironmentAware {
    private Set<String> kafkaTopicsDisabledSet = new HashSet<>();
 
    // BSM
+   private final String BSM_START_FLAG = "0014";
```

Comment:
Do we think about moving these flags into a separate Enum or other separate structure? Can we make this public and constant and remove the needs of getters?

Reply:
Going to move this to a generic class for easier header removal in a future backlog item.